src/reducers/custom-wildcard-field.test.ts

```diff
@@ -1,15 +1,15 @@
 import {ExpandedType} from 'compassql/build/src/query/expandedtype';
 import {CUSTOM_WILDCARD_ADD, CUSTOM_WILDCARD_ADD_FIELD, CUSTOM_WILDCARD_MODIFY_DESCRIPTION,
         CUSTOM_WILDCARD_REMOVE, CUSTOM_WILDCARD_REMOVE_FIELD} from '../actions/custom-wildcard-field';
-import {CustomWildcardField} from '../models/custom-wildcard-field';
+import {CustomWildcardFieldDef} from '../models/custom-wildcard-field';
 import {customWildcardFieldReducer} from './custom-wildcard-field';
 
 describe('reducers/custom-wildcard-field', () => {
   describe(CUSTOM_WILDCARD_ADD, () => {
     it('should return a custom wildcard field array containing one custom wildcard field', () => {
-      const noCustomWildcardFields: CustomWildcardField[] = [];
-      const customWildcardFields = customWildcardFieldReducer(
-        noCustomWildcardFields,
+      const noCustomWildcardFieldDefs: CustomWildcardFieldDef[] = [];
```

Comment:
Just use `[]`, the name is unnecessary and is a bit confusing the first time I saw it.